app/jobs/purge_job.rb

```diff
@@ -26,7 +26,7 @@ def perform(*_args)
       attributes = Patient.new.attributes.keys
       attributes -= PurgeJob.attributes_to_keep
       # Set everything else to nil
-      mask = attributes.collect { |a| [a, nil] }.to_h.symbolize_keys
+      mask = attributes.index_with { |_a| nil }.symbolize_keys
```

Comment:
can lines 26-30 be taken out of the loop so they don't need to be repeated?

Reply:
i made some adjustments in https://github.com/SaraAlert/SaraAlert/pull/1101/commits/08590bbc1f0b93e9857bf8051e7886b6d679b03d